Sources/MongoSwift/MongoCredential.swift

```diff
@@ -0,0 +1,67 @@
+/// Represents an authentication credential.
+public struct MongoCredential: Decodable, Equatable {
+    /// A string containing the username. For auth mechanisms that do not utilize a password, this may be the entire
```

Comment:
I personally prefer there were newlines between each of the properties, though I'm not sure if we're consistent about this in the codebase.

Reply:
ditto for `Mechanism` below.